backend/signature/signature-core/build.gradle.kts

```diff
@@ -14,8 +23,46 @@ dependencies {
 
     api(project(":signature:signature-api"))
 
+    implementation("io.quarkiverse.amazonservices:quarkus-amazon-cognito-user-pools")
+    implementation("software.amazon.awssdk:url-connection-client")
     testImplementation(project(":common:common-test"))
+
+    // https://mvnrepository.com/artifact/org.eclipse.microprofile.rest.client/microprofile-rest-client-api
+    implementation("org.eclipse.microprofile.rest.client:microprofile-rest-client-api:4.0")
```

Comment:
How is it used? If only in tests, can you please take a look how it's done in eln-core? It uses Feign client, can we make it work the same?

Reply:
Yes, great! We'd better have common functionality used by all modules